model/model.go

```diff
@@ -66,6 +66,7 @@ type ColumnInfo struct {
 	Offset              int                 `json:"offset"`
 	OriginDefaultValue  interface{}         `json:"origin_default"`
 	DefaultValue        interface{}         `json:"default"`
+	DefaultValueBit     []byte              `json:"default_bit"`
```

Comment:
Is it possible to use a uniform default value in cloumn info?

Reply:
https://docs.google.com/document/d/17REO2MQ5v_AFOBE9ovehZDGlrtvhjCy3S1p7F1HWaSc/edit
This document explains the reason to do this way.

Reply:
It is better to write down how to do it in the description of PR.

Reply:
1. What's the problem
2. How to do it
3. Is there any compatibility problem

Reply:
@ciscoxll done. PTAL

Reply:
Will it be incompatible when rolling update TiDB? 
As follows case:
On new TiDB, we builds a "add column" job , then we handle the DDL job on Old TiDB.

Reply:
@crazycs520

Reply:
No. It is compatible.
if old tidb to handle job that created by new tidb, old tidb will ignore the `DefaultValueBit `.
I have done an experiment in this scene.